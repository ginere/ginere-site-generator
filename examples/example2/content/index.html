<html>
  {widget.html}
</br>
|VARIABLE_NAME|
</html>
