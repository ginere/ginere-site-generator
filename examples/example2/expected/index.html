<html>
  Hello wolrd!
</br>
This is the name of a variable.
This is a new Line, but this is in the same line.
</html>
